corehq/apps/user_importer/helpers.py

```diff
@@ -300,17 +283,15 @@ def update_locations(self, location_codes, membership, domain_info):
                 locations_info = ", ".join([f"{location.name}[{location.location_id}]" for location in locations])
             else:
                 locations_info = []
-            self.logger.add_info(_("Assigned locations: {locations_info}").format(
-                locations_info=locations_info
-            ))
+            self.logger.add_info(f"Assigned locations: {locations_info}")
 
         # log this after assigned locations are updated, which can re-set primary location
         user_updated_primary_location_id = get_user_primary_location_id(self.user, self.user_domain)
         if user_updated_primary_location_id != user_current_primary_location_id:
             if user_updated_primary_location_id:
                 self._log_primary_location_info()
             else:
-                self.logger.add_info(_("Primary location: None"))
+                self.logger.add_info("Primary location: None")
```

Comment:
already present
https://github.com/dimagi/commcare-hq/commit/c7e498725df7a04c5b8133fbbaa750c6fbe3486b#diff-c9557c19413e04ae10a25832b30f73ee68d4e9bdae31b49f5404b9086b80f126R32324

Reply:
actually this was newly added added but a fuzzy match populated
its surprising, like password reset, even the en file has
```
#: corehq/apps/user_importer/helpers.py:313
#, fuzzy
msgid "Primary location: None"
msgstr "Casos IRA Tratados"
```

Reply:
en
```
#: corehq/apps/user_importer/helpers.py:313
#, fuzzy
msgid "Primary location: None"
msgstr "Casos IRA Tratados"
```

fra
```
#: corehq/apps/user_importer/helpers.py:313
#, fuzzy
#| msgid "Primary Location"
msgid "Primary location: None"
msgstr "Site principal"
```

por
```
#: corehq/apps/user_importer/helpers.py:313
#, fuzzy
#| msgid "Primary Location"
msgid "Primary location: None"
msgstr "Localização Primária"
```

es
```
#: corehq/apps/user_importer/helpers.py:313
#, fuzzy
#| msgid "Primary Location"
msgid "Primary location: None"
msgstr "Ubicación Principal"
```

no translation in other langs